src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -0,0 +1,178 @@
+import * as Yup from "yup";
+
+import { createUrlSafeName, VenueInput, PlacementInput } from "api/admin";
+import firebase from "firebase/app";
+import "firebase/functions";
+import {
+  PLAYA_VENUE_SIZE,
+  MAX_IMAGE_FILE_SIZE_BYTES,
+  GIF_RESIZER_URL,
+  PLAYA_WIDTH,
+  PLAYA_HEIGHT,
+} from "settings";
+
+const initialMapIconPlacement: VenueInput["placement"] = {
```

Comment:
The naming conventions for consts usually states that they should have their names in form of `INITIAL_MAP_ICON_PLACEMENT`